src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -153,7 +153,7 @@ public PerformanceAnalyzerPlugin(final Settings settings, final java.nio.file.Pa
         OpenSearchResources.INSTANCE.setSettings(settings);
         OpenSearchResources.INSTANCE.setConfigPath(configPath);
         OpenSearchResources.INSTANCE.setPluginFileLocation(
-                new Environment(settings, configPath).pluginsFile().toAbsolutePath().toString()
+                new Environment(settings, configPath).pluginsDir().toString()
```

Comment:
Why are we changing FileLocation with directory here ?

Reply:
I would avoid such changes in a version upgrade PRs, hard to track regressions from such PRs.
But why this change ?

Reply:
```Environment.class``` that comes from OpenSearch 3.0 has the pluginsFile() method removed and replaced with pluginsDir() and hence this change was done as part of version upgrade due to build failure.